spiritualoutfitters.html

```diff
@@ -0,0 +1,218 @@
+                            data="https://content.crossroads.net/framework/thirdparty/tinymce/plugins/media/moxieplayer.swf"
+                            type="application/x-shockwave-flash">
+                            <param name="src"
+                                value="https://content.crossroads.net/framework/thirdparty/tinymce/plugins/media/moxieplayer.swf">
+                            <param name="flashvars"
+                                value="url=https%3A//s3.amazonaws.com/crds-cms-uploads/media/video/GoSomewhere-loop-crf30.mp4&amp;poster=/">
+                            <param name="allowfullscreen" value="true">
+                            <param name="allowscriptaccess" value="true">
+                        </object>
+                    </video>
+                </div>
+            </div>
+            <div class="col-sm-8 col-sm-offset-2 text-center">
+                <img class="push-top"
+                    src="https://images.ctfassets.net/y3a9myzsdjan/1diMucwFYgSS6VijCOORpT/14ec6801614bcaff084ad6d95b6ca2f4/crsrds-badge-2020.png">
+                <p class="text-primary font-family-serif push-ends soft-ends">
+                    Since the dawn of social media, the shorthand for Crossroads has been “CRDS”. That’s pronounced
+                    “curds”, like cheese curds. Which is gross. A brand refresh meant we could take another swing at the
+                    shorthand game. We landed on “CRSRDS”, which is pronounced “Crossroads”. Turns out the guy who came
+                    up with CRDS was from Wisconsin, his resume is trending on ZipRecruiter.
+                </p>
+            </div>
+        </div>
+    </div>
+</section>
+
+<section style="background-color: #dbd1bf;">
+    <div class="container-fluid">
+        <div class="row">
+            <div class="col-sm-8 col-sm-offset-2 text-center mt-100 mb-100">
+                <h1 class="text-orange text-uppercase flush-bottom font-family-base-black mt-100">
+                    let's do<br>this thing
+                </h1>
+                <p class="text-primary text-center font-family-serif soft-ends">
+                    Crossroads isn’t going anywhere. We’re doubling down on our mission to partner with God in his
+                    effort to make earth look more like heaven. Whether that’s in our backyard, our neighborhood, our
+                    nation, or our world. We’re committed, now more than ever, to be a training ground for those who
+                    want to take a swing at making a difference. You might say that Crossroads exists to equip people
+                    for the fight. We call it being a spiritual outfitter for those on the adventure of faith.<br>
+                    <span class="font-family-base-bold">It’s going to be a wild ride.</span>
+                </p>
+                <img
+                    src="https://images.ctfassets.net/y3a9myzsdjan/66x1B6FptzkzErSolzYXgn/313eec4a1f21b80551c150ff0d2353f7/x-badge-2020.png">
```

Comment:
If its not a GIF, it should be using thew imgix endpoint.

Reply:
Ok those should be good to go now, thanks!